### PR TITLE
added all the pages

### PR DESCRIPTION
**Pull request by haarsh24**

### CODE REVIEW

File: pages/quiz.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+ 
+
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="../styles/main.css" />
+    <title>SneakerXclub</title>
+
+</head>
+
+<body>
```
Comment (rohanmathur91) [inline pages/quiz.html:17]: Avoid spacing

File: pages/quiz.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://litui.netlify.app/components/components-style.css">
```
Comment (ravikumar0403) [inline pages/quiz.html:7]: You can add meta author tag

File: pages/quiz.html [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+        </div>
+        
+        <div class="nav-icon">
+            <a href="./login.html" class="h3 p1"> Login</a>
+        
+        </div>
+    </div>
+
+
+    <div class="">
```
Comment (rohanmathur91) [inline pages/quiz.html:31]: remove the empty class

Comment (ravikumar0403) [inline pages/quiz.html:31]: you can remove this class attritube if no class is applied

File: styles/index.css [hunk 4]
```diff
@@ -14,5 +14,77 @@
...
\ No newline at end of file
+
+}
+.input-search {
+    margin: 0;
+    width:15rem;
+  } 
+  .header{
+    margin-bottom: 0;
+    /* padding-bottom: 0; */
```
Comment (rohanmathur91) [inline styles/index.css:25]: remove commented code if not needed

File: pages/single-quiz.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+            </li>
+            <li class="lists-item-stacked  p2">
+              I'll tell you how I became the prince of a town called Bel-Air
+            </li>
+            <li class="lists-item-stacked  p2">
+              And I like to take a minute and sit right here
+            </li>
+            
+          </ul>
+          <button class="btn quiz-btn m1"><a href="./quiz.html">Next</a> </button>
```
Comment (rohanmathur91) [inline pages/single-quiz.html:53]: Only use `anchor tag` if want to navigate, and apply the styles of button to it

File: pages/quiz.html [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+                </li>
+                <li class="lists-item-stacked  p2">
+                  I'll tell you how I became the prince of a town called Bel-Air
+                </li>
+                <li class="lists-item-stacked  p2">
+                  And I like to take a minute and sit right here
+                </li>
+                
+              </ul>
+              <button class="btn quiz-btn m1"><a href="../index.html">Finish</a> </button>
```
Comment (ravikumar0403) [inline pages/quiz.html:141]: you should not use a tag inside button tag, you can make you button styles such that it overrides `a` tag's default styles in order to make it look like button

**Review by rohanmathur91 — APPROVED** [review]
Suggestion: Use any tool or extension for prettifying code

File: styles/index.css (hunk 4 above)
Comment (ravikumar0403) [inline styles/index.css:25]: remove commented code if not required

**Review by ravikumar0403 — APPROVED** [review]
Code is clean and easy to understand. I've left a few sanity comments

**Pull request merged**